packages/pycharm-community-bin/PyCharm.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Icon=/usr/src/pacstall/pycharm-community-bin/usr/bin/pycharm-community/bin/pycharm.svg
```

Comment:
Why would an image file be in a bin directory?

Reply:
This is something the PyCharm devs have decided. I have moved the icon reference to the suggested `/usr/share/icons` directory.